.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,18 @@
+rebaseMergeAllowed: true
+squashMergeAllowed: true
+mergeCommitAllowed: false
+branchProtectionRules:
+- pattern: master
+  isAdminEnforced: true
+  requiredStatusCheckContexts:
+    - 'ci/circleci: unit-python2.7'
```

Comment:
should we consider removing python2.7 as that is deprecated/not supported in general?

Reply:
Totally up to you :) I would suggest landing this as is, and then coming back around to remove any python 2.7 stuff after the fact if needed

Reply:
Sure. SGTM.